README.md

```diff
@@ -66,7 +66,7 @@ $ npm install inversify reflect-metadata --save
 
 The InversifyJS type definitions are included in the inversify npm package.
 
-> :warning: **Important!** InversifyJS requires TypeScript >= 2.0 and the `experimentalDecorators`, `emitDecoratorMetadata`, `types` and `lib`
+> :warning: **Important!** InversifyJS requires TypeScript >= 4.4 because support for or `string` | `symbol` indexable objects were added, and the `experimentalDecorators`, `emitDecoratorMetadata`, `types` and `lib`
```

Comment:
you can omit the explanations because other this were added too like strict null checks.
just modify 2.0 to 4.4 and it's fine